cmd/nerdctl/run.go

```diff
@@ -138,6 +138,7 @@ func setCreateFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("hostname", "h", "", "Container host name")
 	// #endregion
 
+	cmd.Flags().String("ipc", "", "IPC namespace to use")
```

Comment:
Could you add shell completion?  Just like `cmd.RegisterFlagCompletionFunc("cgroupns", ...)` in L153-154

Reply:
Done! Sorry completely missed this :+1: